### ISSUE TITLE
[Max] 'deactivate other object' link relationship doesn't work

### ISSUE THREAD

**Issue by smallg2020** [TheGameCreators/GameGuruRepo#4649]
![image](https://github.com/TheGameCreators/GameGuruRepo/assets/66390691/24ffa0c7-573c-4a41-980d-4f4d7a7a3cec)

not sure if this is supposed to be supported yet or not but PerformLogicConnections(e) with this setting doesn't remove the linked object(s) from the game

**Comment by LeeBamberTGC** [comment]
@smallg2020 The PerformLogicConnections command resides in the LUA script and will never affect any links in the level design for connecting two objects. Connections can exist in the level edit, but the PerformLogicConnections only works on those links, and does not remove them.

**Comment by smallg2020** [comment]
yes, i think my understanding of it is that it would do the opposite of the normal trigger zone - i.e. if you set an object to spawnatstart=no and walk into a zone it will spawn / activate the object
so if you were to set this "deactivate other object" i would expect it to hide the object but by default the option appears to set g_Entity[e].activated = 0 and as this is true for most objects by default the option doesn't really do anything by default.

i think it's just a case of not having the same understanding of what the behaviour should do.

**Comment by smallg2020** [comment]
nah, i think it's ok as is if that's what you were intending the link to do, just thought it was meant to destroy the linked entity in game (i.e. more like the opposite of a spawn trigger), i'm not really sure what the current use is for it, is there a stock behaviour or demo that makes use of the setting?

**Comment by LeeBamberTGC** [comment]
@smallg2020 It's just the opposite of the activate trigger. It would be used if you had a switch that was activated, and when you enter a zone it switches the switch back off, that sort of thing :)

**Comment by LeeBamberTGC** [comment]
P.S. I have switched this issue to a feature request in case you want to add some ideas for extra options in the UI here :)
